packages/private/edtr-io/src/plugins/equations/editor.tsx

```diff
@@ -319,7 +339,10 @@ function StepEditor(props: StepEditorProps) {
   return (
     <>
       <LeftTd
-        onClick={() => gridFocus.setFocus({ row, column: StepSegment.Left })}
+        tabIndex={
+          gridFocus.isFocused({ row, column: StepSegment.Left }) ? -1 : 0
```

Comment:
That looks like a big anti-pattern. I'm pretty sure that you shouldn't modify `tabIndex` dynamically.